docs/guide/persistence_cache.md

```diff
@@ -108,7 +97,23 @@ services:
 
 ##### Redis Cluster
 
-It is possible to set up and use Redis as a cluster. This configuration is more efficient and reliable for large installations. Redis Cluster can be configured in two ways, the first using [create-cluster script](https://redis.io/topics/cluster-tutorial) and the second using [Redis Sentinel](https://redis.io/topics/sentinel). If you use Platform.sh Enterprise you can benefit from the Redis Sentinel across three nodes for greater fault tolerance. Platform.sh Professional and lower versions offer Redis in single instance mode only. Configuration on eZ Platform / Symfony stays the same regardless of the Redis version, single instance mode or cluster mode.
+Configuration of several Redis instances can be done in several ways, as Persistence cache depends on all involved web servers seeing the same view of the cache as it's shared, the following applies:
+- [Redis Cluster](https://redis.io/topics/cluster-tutorial)
+  - Shards cache across several instances in order to be able to cache more then memory of one server allows
+  - Shard slaves can improve availability, however these are eventually consistent so can not be used for reads
```

Comment:
Rather a question, I think, that slaves can be safely used for reads, but maybe I'm missing something?

Reply:
Are they consistent with the data on masters?

But you are right, I need to dig up sources for these claims, so I'll double check that while on it.